types/ckeditor__ckeditor5-table/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
```

Comment:
Is the target necessary?  I wouldn't think it would make a difference for .d.ts files.

Reply:
Done! :+1: